api/v1alpha1/addon_types.go

```diff
@@ -69,6 +71,9 @@ type Addons struct {
 
 	// +optional
 	CSIProviders *CSI `json:"csi,omitempty"`
+
+	// +optional
+	ServiceLoadbalancer *ServiceLoadbalancer `json:"serviceLoadbalancer,omitempty"`
```

Comment:
nit: Kubernetes uses "LoadBalancer" or "load balancing"
```suggestion
	ServiceLoadBalancer *ServiceLoadBalancer `json:"serviceLoadBalancer,omitempty"`
```

Reply:
Fixed all these up in 410c70174b0968e7a5d77531b6a37d6c5fa0816d